datafusion/core/src/datasource/statistics.rs

```diff
@@ -35,6 +35,7 @@ pub async fn get_statistics_with_limit(
     all_files: impl Stream<Item = Result<(PartitionedFile, Statistics)>>,
     file_schema: SchemaRef,
     limit: Option<usize>,
+    collect_stats: bool,
```

Comment:
Given the function is called 'get_statistics` and now there is a boolean flag that implies the stats aren't collected, can we please add some documentation to the docstring explaining what the `collect_stats` flag does?

Reply:
Yes good catch, will do.